usaspending_api/api_contracts/contracts/v2/bulk_download/awards.md

```diff
@@ -25,8 +25,8 @@ This route sends a request to the backend to begin generating a zipfile of award
             {
                 "filters": {
                     "agency": 50,
-                    "prime_award_types": ["contracts", "grants"],
-                    "sub_award_types": ["procurement"],
+                    "prime_award_types": ["02", "03", "04", "05", "A", "B", "C", "D"],
+                    "sub_award_types": ["procurement"]
```

Comment:
Missing a comma at the end of this line

Reply:
Good catch! That is what I get for copy/paste.